lncfg/config.go

```diff
@@ -0,0 +1,82 @@
+package lncfg
+
+import (
+	"time"
+
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+type ChainConfig struct {
```

Comment:
can each subconfig get its own file? the `Config` suffixes can also be dropped

Reply:
Right, makes sense. Added a godoc comment to each struct as well.